src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,108 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private final float loadFactor;
+    private final int scaleFactor;
+    private int size;
+    private int capacity;
+    private Node<K, V> [] table;
+
+    public MyHashMap() {
+        this(16, .75f);
```

Comment:
16 and .75f are magic numbers, create constants
```suggestion
        this.table = (Node<K, V>[]) new Node[capacity];
```